jsonrpc.go

```diff
@@ -451,11 +451,16 @@ func (rpcResponse *RPCResponse) GetObject(toType interface{}) error {
 		return err
 	}
 
-	err = json.Unmarshal(js, toType)
+	err = json.Unmarshal(js, &toType)
+
 	if err != nil {
 		return err
 	}
 
+	if toType == nil {
```

Comment:
This can only happen if you provide a nil pointer as input. But then you should also expect a nil as ouput that is the go aquivalent of json null.

I don't know if this should be an error since:
1. the request you send to the rpc server was successfull
2. the response was null
This seems ok and should not throw an error, should it?

Reply:
Consider this:

```go
p := &Person{}
err := result.GetObject(p)
```

You won't get an error, even if the response from the server was:
`{"jsonrpc":"2.0","result":null,"id":1}`